terraform/modules/happy-stack-eks/main.tf

```diff
@@ -231,6 +231,11 @@ module "tasks" {
   platform_architecture = each.value.platform_architecture
   is_cron_job           = each.value.is_cron_job
   cron_schedule         = each.value.cron_schedule
-  additional_env_vars   = merge(local.db_env_vars, var.additional_env_vars, local.stack_configs)
+
+  additional_env_vars                  = merge(local.db_env_vars, var.additional_env_vars, local.stack_configs)
```

Comment:
We might want to group `additional_env_vars*` from all sources and `additional_volumes_from*` into objects eventually.